src/ic-cdk/src/api/management_canister/http_request/mod.rs

```diff
@@ -39,14 +39,14 @@ thread_local! {
 #[export_name = "canister_query <ic-cdk internal> http_transform"]
 extern "C" fn http_transform() {
     use crate::api::{
-        call::{arg_data, reply},
+        call::{arg_data, reply, ArgDecoderConfig},
         caller,
     };
     if caller() != Principal::management_canister() {
         crate::trap("This function is internal to ic-cdk and should not be called externally.");
     }
     crate::setup();
-    let (args,): (TransformArgs,) = arg_data();
+    let (args,): (TransformArgs,) = arg_data(ArgDecoderConfig::default());
```

Comment:
Is there anything a developer needs to know about the use of this default here?

Reply:
Isn't the TransformArgs arbitrarily large?  While we trust the management canister, the response and the context don't come from directly from it, right?

```
pub struct TransformArgs {
    /// Raw response from remote service, to be transformed
    pub response: HttpResponse,

    /// Context for response transformation
    #[serde(with = "serde_bytes")]
    pub context: Vec<u8>,
}
```

Reply:
Yes, we only set the skipping quota here. The decoding of expected types, i.e., HttpResponse and blob can be arbitrarily large, and doesn't count towards the skipping quota. Same as the default setting with canister endpoint, we don't set a limit for what's being requested (expected type) by the code. We only want to reject malicious messages which contain too many unneeded information.